src/kitchen/package.json

```
@@ -1,10 +1,6 @@
 {
-  "name": "@guardian/src-ed-star-rating",
-  "version": "3.9.0",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/guardian/source.git"
-  },
+  "name": "@guardian/source-kitchen-react",
```

what do you think about how about this? 
```suggestion
  "name": "@guardian/react-components-development-kitchen",
```
it fits better if we end up going with `@guardian/source` and `@guardian/react-components`.

i also think having a clunky old name like that implies more that this is not a finished product to the unwitting